qiskit_experiments/library/randomized_benchmarking/__init__.py

```diff
@@ -31,13 +32,18 @@
 
 .. autosummary::
     :toctree: ../stubs/
+    :template: autosummary/analysis.rst
 
     RBAnalysis
     InterleavedRBAnalysis
+
+.. autosummary::
+    :toctree: ../stubs/
+
     RBUtils
 """
-from .rb_experiment import StandardRB
+from .interleaved_rb_analysis import InterleavedRBAnalysis
 from .interleaved_rb_experiment import InterleavedRB
 from .rb_analysis import RBAnalysis
-from .interleaved_rb_analysis import InterleavedRBAnalysis
+from .rb_experiment import StandardRB
```

Comment:
Does the order of the imports matter?

Reply:
No, my IDE ran auto import optimize. Perhaps it's aligned in alphabetical ordering.

Reply:
I'll revert this if it matters.